catalog/opensearch/catalog-opensearch-source/pom.xml

```diff
@@ -165,10 +164,20 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>ddf.platform.util</groupId>
+            <artifactId>platform-util</artifactId>
```

Comment:
The version is in `dependencyManagement`; if the version is updated there, i believe it will be propagated to all sub modules.  https://github.com/codice/ddf/blob/master/catalog/pom.xml#L201